functests/ptp/ptp.go

```diff
@@ -124,12 +135,16 @@ var _ = Describe("ptp", func() {
 
 	var _ = Describe("Test Offset", func() {
 		BeforeEach(func() {
-			nodes, err := client.Client.Nodes().List(metav1.ListOptions{
-				LabelSelector: ptpSlaveNodeLabel,
-			})
-			Expect(err).ToNot(HaveOccurred())
-			if len(nodes.Items) < 2 {
-				Skip(fmt.Sprintf("PTP Nodes with label %s are not deployed on cluster", ptpSlaveNodeLabel))
+			if discoveryMode {
```

Comment:
Let's do the other way around, so checkPolicies works also for the local setup

```
if !discoveryMode {
   // Do the setup
}

checkPolicies()
```

Reply:
Done